aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/forms/ControlRow.get.js

```diff
@@ -0,0 +1,70 @@
+model.jsonModel = {
+   services: [
+      {
+         name: "alfresco/services/LoggingService",
+         config: {
+            loggingPreferences: {
+               enabled: true,
+               all: true,
+               warn: true,
+               error: true
+            }
+         }
+      }
+   ],
+   widgets: [
+      {
+         id: "FORM1",
+         name: "alfresco/forms/Form",
+         config: {
+            pubSubScope: "FORM1_",
+            widgets: [
+               {
+                  id: "CR1",
+                  name: "alfresco/forms/ControlRow",
+                  config: {
+                     widgets: [
+                        {
+                           id: "SELECT1",
+                           name: "alfresco/forms/controls/Select",
+                           config: {
+                              fieldId: "SELECT1",
+                              name: "select",
+                              label: "Select option",
+                              description: "Selecting an option should update the dynamic payload button",
+                              optionsConfig: {
+                                 fixed: [
+                                    { label: "One", value: "ONE" },
+                                    { label: "Two", value: "TWO" }
+                                 ]
+                              }
+                           }
+                        }
```

Comment:
Am I missing something, or does this test only include a single form control? Is there anything to be gained by validating that the ControlRow can live up to it's purpose of rendering 'multiple form controls on a single row'? E.g. you could ensure that the values are published for items that aren't the first in the list.